### PR TITLE
Update 31-to-50.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnetcore/migration/31-to-50.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ uid: migration/31-to-50
 
 By [Scott Addie](https://github.com/scottaddie)
```
Comment (Rick-Anderson) [inline aspnetcore/migration/31-to-50.md:13]:
```suggestion
```

File: aspnetcore/migration/31-to-50.md [hunk 2]
```diff
@@ -10,9 +10,8 @@ uid: migration/31-to-50
 ---
 # Migrate from ASP.NET Core 3.1 to 5.0
 
-By [Scott Addie](https://github.com/scottaddie)
 
-This article explains how to update an existing ASP.NET Core 3.1 project to ASP.NET Core 5.0.
+This article explains how to update an existing ASP.NET Core 3.1 project to ASP.NET Core 5.0. For instructions on how to migrate from ASP.NET Core 3.1  to ASP.NET Core 6.0, see <xref:migration/31-to-60>.
```
Comment (serpent5) [inline aspnetcore/migration/31-to-50.md:14]:
```suggestion
This article explains how to update an existing ASP.NET Core 3.1 project to ASP.NET Core 5.0. For instructions on how to migrate from ASP.NET Core 3.1 to ASP.NET Core 6.0, see <xref:migration/31-to-60>.
```

File: aspnetcore/migration/50-to-60.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ uid: migration/50-to-60
 ---
 # Migrate from ASP.NET Core 5.0 to 6.0
 
-This article explains how to update an existing ASP.NET Core 5.0 project to ASP.NET Core 6.0.
+This article explains how to update an existing ASP.NET Core 5.0 project to ASP.NET Core 6.0. For instructions on how to migrate from ASP.NET Core 3.1  to ASP.NET Core 6.0, see <xref:migration/31-to-60>.
```
Comment (serpent5) [inline aspnetcore/migration/50-to-60.md:13]:
```suggestion
This article explains how to update an existing ASP.NET Core 5.0 project to ASP.NET Core 6.0. For instructions on how to migrate from ASP.NET Core 3.1 to ASP.NET Core 6.0, see <xref:migration/31-to-60>.
```

**Review by serpent5 — APPROVED** [review]
LGTM - there's just a cheeky extra space hiding in there.

**Pull request merged**